packages/manager/modules/nutanix/src/dashboard/component/uninstall-modal/translations/Messages_fr_FR.json

```diff
@@ -1,8 +1,7 @@
 {
   "nutanix_dashboard_uninstall_node_title": "Désinstaller le noeud",
-  "nutanix_dashboard_uninstall_node_description": "Voulez-vous vraiment éteindre votre noeud \"{{serviceName}}\"? La désinstallation d'un noeud Nutanix peut avoir un impact sur votre cluster, vérifier les pré-requis sur le portail de Nutanix pour réaliser cette action",
+  "nutanix_dashboard_uninstall_node_description": "La désinstallation d'un noeud Nutanix peut avoir un impact sur votre cluster, vérifier les pré-requis sur le portail de Nutanix pour réaliser cette action",
```

Comment:
```suggestion
  "nutanix_dashboard_uninstall_node_description": "La désinstallation d'un noeud Nutanix peut avoir un impact sur votre cluster, vérifiez les pré-requis sur le portail de Nutanix pour réaliser cette action",
```

Reply:
Seen with Anta and Raissa, `vérifier` is ok.